tests/bluechi_test/bluechictl.py

```diff
@@ -51,6 +51,23 @@ def get_status(self, node_name: str, unit_name: str, check_result: bool = True,
             expected_result
         )
 
+    def get_node_status(self, node_name: str = None, check_result: bool = True, expected_result: int = 0) \
+            -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+        if node_name:
+            return self._run(
+                f"Getting status of node '{node_name}'",
+                f"status {node_name}",
+                check_result,
+                expected_result
+            )
+        else:
+            return self._run(
+                "Getting status of all nodes",
+                "status",
+                check_result,
+                expected_result
+            )
```

Comment:
I'd remove the `else` here:
```suggestion
            return self._run(
                "Getting status of all nodes",
                "status",
                check_result,
                expected_result
            )
```

Reply:
Done